home/mon_data.asm

```diff
@@ -31,24 +31,24 @@ GetBaseData::
 	jr .end
 
 .egg
-; ????
 	ld de, UnknownEggPic
```

Comment:
Wasn't this also a gen1 thing?

Reply:
Gen 1 wouldn't have had to special-case Egg.

Reply:
Right shit yeah :sweat_smile: